ecommerce/certprep/static/sass/partials/components/_navbar.scss

```diff
@@ -0,0 +1,152 @@
+// navbar
```

Comment:
Do you need all these definitions? can we remove some unnecessary rules? @sergivalero20

Reply:
Done!
I removed unnecessary rules.

Reply:
This file should be moved to the root sass folder, please check this definition: https://github.com/Pearson-Advance/openedx-themes/blob/open-release/juniper.stage/ecommerce/regis/static/sass/base/main.scss @sergivalero20